src/UIComponents/ResultsTable.jsx

```diff
@@ -25,61 +25,65 @@ class ResultsTable extends Component {
     const featureCount = this.props.selectedFeatures.length;
 
     return (
-      <div style={styles.scrollableContents}>
-        <table>
-          <thead>
-            <tr>
-              <th colSpan={featureCount} style={styles.largeText}>
-                Features
-              </th>
-              <th>
-                <span style={styles.largeText}>{"A.I. Prediction"}</span>
-              </th>
-              <th>
-                <span style={styles.largeText}>{"Actual"}</span>
-                {this.props.isRegression && (
-                  <div style={styles.smallText}>{"+/- 3% of range"}</div>
-                )}
-              </th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr>
-              {this.props.selectedFeatures.map((feature, index) => {
+      <div style={styles.panel}>
+        <div style={styles.tableParent}>
+          <table style={styles.dataDisplayTable}>
+            <thead>
+              <tr>
+                <th colSpan={featureCount} style={{...styles.largeText, ...styles.dataDisplayHeader, ...styles.resultsTableFirstHeader }}>
```

Comment:
You can use [Prettier](https://prettier.io/) to get the formatting looking just right.

Reply:
Thanks for the tip! I pushed changes based off your comments and updated some of the style names.